framework/include/materials/Material.h

```diff
@@ -1,3 +1,4 @@
+
 //* This file is part of the MOOSE framework
```

Comment:
```suggestion
//* This file is part of the MOOSE framework
```

Please remove this newline

Reply:
Oops, I added it accidentally.